typhos/display.py

```diff
@@ -1110,11 +1115,17 @@ def _generate_template_menu(self, base_menu):
                 view = view.split('_screen')[0]
             menu = base_menu.addMenu(view.capitalize())
 
+            duplicates = utils.find_duplicate_filenames_in_paths(filenames)
+
             for filename in filenames:
+                current_filename = os.path.split(filename)[-1]
+
                 def switch_template(*, filename=filename):
                     self.force_template = filename
-
-                action = menu.addAction(os.path.split(filename)[-1])
```

Comment:
The old code could have used `os.path.basename(filename)` instead of picking the last (second) element from the path split, and the new code can do this too since you don't need the directory name

Reply:
good one, added.